Sources/SwifterSwift/UIKit/UILabelExtensions.swift

```diff
@@ -34,6 +34,40 @@ public extension UILabel {
         label.sizeToFit()
         return label.frame.height
     }
+    
+    
+    /// SwifterSwift: Add attribute to UILabel
+    /// - Parameters:
+    ///   - range: The range of the string to apply attributes
+    func attribute(textIn range: Range<String.Index>?, _ attributes: [NSAttributedString.Key : Any]) {
```

Comment:
I think a better interface would be something like `setAttributes(_ attributes: [NSAttributedString.Key : Any], inRange: Range<String.Index>)`

Reply:
Right, okay. Thanks!